### DISCUSSION TITLE
夸张的旧版本下载

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#2786] [提问] [answered]
我就是随便用个1.7.2尝试复现 #2781 他给我写了7000多行日志这合理吗
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/12909786/Log1.txt)

**Comment by kong-que-dnf123** [comment]
不是我就又用了一次1.17.1下载怎么还出了个没错误窗口的错误提示呢（虽然1.17.1已经下载完了）
 ~要不是调试模式显示了我还真不知道~
![image](https://github.com/Hex-Dragon/PCL2/assets/120994373/fcaeb778-e1ad-4880-9f2a-f280d0c092ea)
[Log1.txt](https://github.com/Hex-Dragon/PCL2/files/12909807/Log1.txt)

**Reply by DreamUniverse843** [reply]
网络请求有重试次数的吧，你第一次失败可能第二次成功了就没弹窗？

**Comment by DreamUniverse843** [comment]
> 我就是随便用个1.7.2尝试复现 #2781 他给我写了7000多行日志这合理吗
> [Log1.txt](https://github.com/Hex-Dragon/PCL2/files/12909786/Log1.txt)

多数都是资源文件下载日志啊，问题应该不大吧😂跟开了调试模式有关系么？
白天有课，晚上回去试试

**Comment by DreamUniverse843** [comment, marked as answer]
好了确定了，源代码里面就这么写的，每一个资源文件都会出日志（不是 bug）
https://github.com/Hex-Dragon/PCL2/blob/037e6b19b46eea98b88664474c0963d44b99caa2/Plain%20Craft%20Launcher%202/Modules/Base/ModNet.vb#L1058
~~这不应该算是问题了~~

**Reply by kong-que-dnf123** [reply]
谢了awa
